specification/cognitiveservices/data-plane/ContentModerator/stable/v1.0/ContentModerator.json

```diff
@@ -3440,8 +3459,8 @@
       "in": "body",
       "required": true,
       "schema": {
-        "format": "binary",
-        "type": "string"
+        "type": "object",
+        "format": "file"
```

Comment:
Just to be sure it's clear to you, that's a breaking change. Before, it was taking a string in memory, now it's taking a file descriptor.
If you expect this to be possibly massive file as input, it makes sense, but still breaking and samples will have to be updated.

Reply:
I will update the sample. I had to change this as autorest generated sdk files for string type text content serializes the string content . For example “profanity term” string gets serialized as "\"profanity term\""
Which causes the profanity term index(position) response from sdk to alter by +1 position(ex: index for this profanity text should be 0 but sdk returns 1).
So I was suggested this change to avoid manual change of sdk files.

Reply:
I discussed this with sdk code generator team and apparently if the type is "string", the assumption is that it needs to be deserialized. And they don't have a way to say "not to deserialize that" . So its not a bug from their end. Regarding other languages, all languages support stream so shouldn't be a problem.I got this verified with their team.